presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -877,6 +877,15 @@ public void testStructSchemaEvolution()
         assertQuery("SELECT * FROM test_struct_add_column", "SELECT (1, 2, 3) UNION ALL SELECT (1, 2, null)");
     }
 
+    @Test
+    public void testOptimizedSubscriptExpression()
+    {
+        getQueryRunner().execute("CREATE TABLE test_varchar_maps AS " +
```

Comment:
We need to use unique table names to allow tests to run concurrently. To make sure table names don't repeat, we tend to use names that match the test name. Here, it would be `test_optimized_subscript_expression`.

Reply:
added the test in planner